shortcuts.yaml

```diff
@@ -1,26 +1,29 @@
 short options:
-'-t': '--authType, --targetTypes, --to, --type, --title, --fieldTitle, --targetUrl, --listTitle, --sectionTemplate, --text, --webTemplate'
-'-u': '--userName, --usageGuidelinesUrl, --appCatalogUrl, --webUrl, --url, --siteUrl'
-'-p': '--password, --path, --properties, --filePath, --policy, --clientSideComponentProperties, --pageSize, --parentFolderUrl, --principals, --promoteAs, --selectProperties, --parentWebUrl'
+'-t': '--authType, --targetTypes, --to, --title, --targetUrl, --sectionTemplate, --text'
+'-u': '--userName, --usageGuidelinesUrl, --appCatalogUrl, --url'
+'-p': '--password, --path, --properties, --filePath, --policy, --clientSideComponentProperties, --parentFolderUrl, --principals, --promoteAs, --parentWebUrl'
 '-c': '--certificateFile, --packageCreatedBy, --classifications, --channelId, --contentTypeId, --clientSideComponentId, --contentType, --clientState, --column, --classification, --content, --comment'
-'-i': '--clientId, --grantId, --appId, --id, --templateId, --groupId, --teamId, --position, --hubSiteId, --includeAssociatedSites, --listId, --requestId, --siteDesignId, --runId, --taskId'
+'-i': '--clientId, --grantId, --id, --templateId, --teamId, --position, --hubSiteId, --includeAssociatedSites, --requestId, --siteDesignId, --runId, --taskId'
 '-r': '--resourceId, --role, --origin, --required, --rights, --resource'
-'-s': '--scope, --packageSourceEnvironment, --subject, --siteUrl, --sortOrder, --sourceUrl, --pageSize, --systemUpdate, --section, --siteScripts'
-'-n': '--displayName, --name, --packageDisplayName, --userName, --notificationUrl, --pageNumber, --pageName'
+'-s': '--packageSourceEnvironment, --subject, --siteUrl, --sourceUrl, --systemUpdate, --section, --siteScripts'
+'-n': '--displayName, --name, --packageDisplayName, --notificationUrl, --pageNumber, --pageName'
 '-e': '--environment, --enabled, --commandUIExtension, --expirationDateTime'
-'-d': '--packageDescription, --description, --displayName, --defaultClassification, --date'
-'-f': '--format, --flow, --fieldId, --filter, --folder, --fileUrl, --folderUrl, --forceRefresh, --fields'
+'-d': '--packageDescription, --description, --defaultClassification, --date'
+'-f': '--format, --flow, --fieldId, --folder, --fileUrl, --folderUrl, --forceRefresh, --fields'
 '-m': '--mailNickname, --messageId, --previewImageUrl'
-'-l': '--logoPath, --listTitle, --location, --logoUrl, --listId, --filter, --layoutType, --lcid, --locale'
+'-l': '--logoPath, --location, --logoUrl, --listId, --layoutType, --lcid, --locale'
 '-g': '--guestUsageGuidelinesUrl, --group'
-'-a': '--all, --alias, --previewImageAltText'
+'-a': '--all, --alias, --previewImageAltText, --appId'
 '-j': '--joined'
 '-v': '--toVersion, --value, --version'
 '-h': '--hidden, --hideDefaultThemes'
 '-x': '--xml, --translateX'
-'-w': '--webUrl, --webTemplate'
-'-parserDisabled': ''
+'-w': '--webUrl'
```

Comment:
All URLs should use `-u` for consistency

Reply:
We have commands that have `webUrl` or `siteUrl` and `url` together. Maybe we should rename url to something else more meaningful?

Example:
```
spo file get:
  - '-w, --webUrl <webUrl>'
  - '-u, --url [url]'
  - '-i, --id [id]'
  - '--asString'
  - '--asListItem'
  - '--asFile'
  - '-p, --path [path]'
```